threescale_api/defaults.py

```diff
@@ -314,6 +314,16 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return str(self)
 
+    def __eq__(self, other) -> bool:
+        return (
```

Comment:
The class should be the same because as @pestanko has mentioned in previous comment, there can be two onjects from 2 different classes with same id and system_name.